version/3_2.sh

```diff
@@ -1,18 +1,16 @@
 #!/bin/bash -e
 
-# Install Python 3.2
-sudo apt-get install -y python3.2 python3.2-dev
+apt-get install -y python3.2 python3.2-dev
+cd /u12pyt/python3.2
+python3.2 get-pip.py
 
-# Install virtualenv
-sudo wget https://bitbucket.org/pypa/setuptools/raw/bootstrap/ez_setup.py -O /tmp/ez_setup.py
-sudo python3.2 /tmp/ez_setup.py
-sudo easy_install-3.2 pip
-pip3.2 install virtualenv
+# Installing virtualenv
+cd /u12pyt/python3.2/virtualenv-13.1.2
+python setup.py install
```

Comment:
Here we are using `python` instead of `python3.2` as we are getting the following error.
```
root@59da57ead918:/u12pyt/python3.2/virtualenv-13.1.2# python3.2 setup.py install
/usr/local/lib/python3.2/dist-packages/pkg_resources/__init__.py:88: UserWarning: Support for Python 3.0-3.2 has been dropped. Future versions will fail here.
  warnings.warn(msg)
Traceback (most recent call last):
  File "setup.py", line 10, in <module>
    from setuptools import setup
  File "/usr/local/lib/python3.2/dist-packages/setuptools/__init__.py", line 11, in <module>
    from setuptools.extern.six.moves import filterfalse, map
  File "/usr/local/lib/python3.2/dist-packages/setuptools/extern/__init__.py", line 1, in <module>
    from pkg_resources.extern import VendorImporter
  File "/usr/local/lib/python3.2/dist-packages/pkg_resources/__init__.py", line 1734, in <module>
    register_loader_type(importlib_machinery.SourceFileLoader, DefaultProvider)
AttributeError: 'module' object has no attribute 'SourceFileLoader'
```